### PR TITLE
[Merged by Bors] - feat: updated http tutorial with transforms on inbound side

### PR DESCRIPTION
**Pull request by galibey**

### CODE REVIEW

File: content/docs/tutorials/data-pipeline.md [hunk 1]
```diff
@@ -84,6 +86,42 @@ Consuming records starting 4 from the end of topic 'cat-facts-data'
 {"fact":"Relative to its body size, the clouded leopard has the biggest canines of all animals\u2019 canines. Its dagger-like teeth can be as long as 1.8 inches (4.5 cm).","length":156}
 ```
 
+#### Inbound Connector with JSON to JSON transformation before send out
```
Comment (ajhunyady) [inline content/docs/tutorials/data-pipeline.md:89]: send out -> writing to topic

File: content/docs/tutorials/data-pipeline.md [hunk 2]
```diff
@@ -84,6 +86,42 @@ Consuming records starting 4 from the end of topic 'cat-facts-data'
...
+We can extend our config file to add an additional JSON to JSON transformation to records.
+
+{{<code file="embeds/connectors/catfacts-basic-connector-with-transform.yaml" lang="yaml" copy="true">}}
+
+In this config, we add the field `source` with the static value `http` to every record. Note that if the field 
+already exists, it will not be overwritten.
+
+Before we create the connector we need to add `infinyon/jolt@0.1.0` SmartModule to the cluster.
+This SmartModule uses a domain specific language (DSL) called [Jolt](https://github.com/infinyon/fluvio-jolt), to specify a transformation of input JSON to another shape of JSON data
+
```
Comment (ajhunyady) [inline content/docs/tutorials/data-pipeline.md:100]: Let's download this SmartModule from the [SmartModule Hub](... add link to hub docs)

File: content/docs/tutorials/data-pipeline.md [hunk 3]
```diff
@@ -105,7 +143,7 @@ This SmartModule will do a basic mapping of the JSON input into a SQL statement
 $ fluvio hub download infinyon/json-sql@0.1.0
```
Comment (ajhunyady) [inline content/docs/tutorials/data-pipeline.md:143]: Is this repeated now? It looks like we have it above.

**Review by ajhunyady — COMMENTED** [review]
Nice job! A few comments inline.

File: content/docs/tutorials/data-pipeline.md (hunk 1 above)
Comment (galibey) [reply content/docs/tutorials/data-pipeline.md]: done

File: content/docs/tutorials/data-pipeline.md (hunk 2 above)
Comment (galibey) [reply content/docs/tutorials/data-pipeline.md:102]: done

File: content/docs/tutorials/data-pipeline.md (hunk 3 above)
Comment (galibey) [reply content/docs/tutorials/data-pipeline.md:145]: No. At first occurrence, we add `jolt` to http, then, `json-sql` to sql and `jolt` to sql with a comment that if haven't done that before.

**Review by ajhunyady — APPROVED** [review]
Looks good

**Review by tjtelan — APPROVED** [review]
New section is great

**Pull request closed without merging**